test/common/testvm.py

```diff
@@ -844,6 +844,44 @@ def virEventLoopNativeRun():
         self.eventLoopThread.setDaemon(True)
         self.eventLoopThread.start()
 
+TEST_DOMAIN_XML="""
```

Comment:
It's intimately connected with this file. It's not changeable independently. Realistically if libvirt had an APIish way to configure this, we would use it and not define this as XML.